msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/AppTokenProviderParameters.java

```diff
@@ -23,4 +16,43 @@ public class AppTokenProviderParameters {
     public String claims;
     /// tenant id
     public String tenantId;
+
+    public AppTokenProviderParameters(Set<String> scopes, String correlationId, String claims, String tenantId) {
```

Comment:
@Avery-Dunn - is there value in reviewing these changes ? I.e. did you de-lombok all of this, in which case it's generaetd code and we can take it at face value?

Reply:
For this first PR, yes: 
- Helps to understand what Lombok was doing, and what effects removing it will have
- Demonstrates what the Lombok plugin will produce and confirms it's correct
- Shows what still needs to be manually adjusted, mainly the comments used in API docs

Going forward I don't think so. The next PR will affect a lot more classes, but as long as there are no API changes (as confirmed via RevAPI) and tests pass then it shouldn't require any sort of deep review.